stories/6-Tab.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React, {Fragment} from "react";
+import {storiesOf} from "@storybook/react";
+
+import Tab, {TabItem} from "../src/tab/Tab";
+
+const tabItems: TabItem[] = [
+  {
+    id: "home",
+    children: "Home"
+  },
+  {
+    id: "following",
+    children: "Following"
+  }
+];
+
+storiesOf("Tab", module).add("Tab", () => (
+  <Fragment>
+    <Tab items={tabItems}>{[<div>{"Home"}</div>, <div>{"Following"}</div>]}</Tab>
```

Comment:
Here we would use `TabBody`:

```suggestion
    <Tab items={tabItems}>
      {[
        <TabBody>{"Home"}</TabBody>,
        <TabBody className={"following-tab"}>{"Following"}</TabBody>
      ]}
    </Tab>
```

Reply:
Wouldn't it be better if we handle wrapping with `<div className={classNames("tab-body", className)}>` inside `Tab` as @mucahit did in the first place? Then we can pass the child component directly and won't have to import `TabBody` every time we use `Tab`.